Sources/StreamChat/WebSocketClient/Events/NotificationEvents.swift

```diff
@@ -217,6 +227,7 @@ public struct NotificationRemovedFromChannelEvent: ChannelSpecificEvent {
 class NotificationRemovedFromChannelEventDTO: EventDTO {
     let cid: ChannelId
     let user: UserPayload
+    // This `member` field is equal to the `membership` field in channel query
     let member: MemberPayload
```

Comment:
What if we rename `member -> membership` to be explicit that it's about the current user?

Reply:
then we would have different naming with our docs and other SDKs

Reply:
Notification events are always about the current user so to me it's fine the member data has different naming from the rest of the member events. But I'm fine with both so leaving it up to you 🙂

Reply:
I think naming parity is important so i'll leave it as is